File: LeetCode/1217. Minimum Cost to Move Chips to The Same Position.py
###  KNOWLEDGE IS POWER  ###

class Solution:
    def minCostToMoveChips(self, position: List[int]) -> int:
        odd, even = 0, 0
        for i in position:
            if i % 2 == 0:
                even += 1
            else:
                odd += 1
        return min(odd, even)

File: LeetCode/1217. Minimum Cost to Move Chips to The Same Position.java
/***  KNOWLEDGE IS POWER  ***/

class Solution {
    public int minCostToMoveChips(int[] position) {
        int even = 0 ;
        int odd = 0 ;
        for (int i : position) 
        {
            if (i%2 == 1) 
            {
                odd++;
            } 
            else 
            {
                even++;
            }
        }
        return Math.min(odd , even) ;
    }
}
